<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>189</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QDialogButtonBox" name="ConfirmButtonBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QSlider" name="ExposureSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>62250</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="ExposureLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>50</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Exposure</string>
   </property>
  </widget>
  <widget class="QSlider" name="GainSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="GainLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>50</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Gain</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="AutoExposureBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="AutoGainBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto</string>
   </property>
  </widget>
  <widget class="QLabel" name="ExposureValueLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="GainValueLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
